mptt/filters.py

```diff
@@ -0,0 +1,116 @@
+from django.conf import settings
+from django.contrib.admin import RelatedFieldListFilter
+from django.contrib.admin.options import IncorrectLookupParameters
+from django.contrib.admin.utils import get_model_from_relation
+from django.core.exceptions import ValidationError
+from django.utils.encoding import smart_text
+from django.utils.html import mark_safe
+from django.utils.translation import ugettext_lazy as _
+from django.utils.translation import get_language_bidi
+from django.db.models.fields.related import ForeignObjectRel, ManyToManyField
+
+MPTT_ADMIN_LEVEL_INDENT = getattr(settings, 'MPTT_ADMIN_LEVEL_INDENT', 10)
```

Comment:
I don't think this is required as a setting. Maybe just put it as a class attribute, and if anyone _really_ needs to override it they can subclass?